CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.0.2-dev.0
```

Comment:
Feels a bit weird since we never had a v0.0.1 release 😛 
Thoughts on renaming this to `0.0.1-nullsafety.0`? That way the next release would ideally be a v0.0.1?

Reply:
sure, no problem